lib/src/view/analysis/analysis_screen.dart

```diff
@@ -835,27 +834,123 @@ class ServerAnalysisSummary extends ConsumerWidget {
     final pgnHeaders = ref.watch(
       analysisControllerProvider(options).select((value) => value.pgnHeaders),
     );
+
     return SingleChildScrollView(
       child: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
           AcplChart(options),
-          if (serverAnalysis != null) ...[
-            const SizedBox(height: 16.0),
-            _PlayerStats(Side.white, serverAnalysis.white, pgnHeaders),
-            _PlayerStats(Side.black, serverAnalysis.black, pgnHeaders),
-          ],
+          if (serverAnalysis != null)
+            Padding(
+              padding: const EdgeInsets.all(16.0),
+              child: Table(
+                defaultVerticalAlignment: TableCellVerticalAlignment.middle,
+                columnWidths: const {
+                  0: FlexColumnWidth(1),
+                  1: FlexColumnWidth(1),
+                  2: FlexColumnWidth(1),
+                },
+                children: [
+                  TableRow(
+                    children: [
+                      _SummaryPlayerName(Side.white, pgnHeaders),
+                      const SizedBox(width: 0),
+                      _SummaryPlayerName(Side.black, pgnHeaders),
+                    ],
+                  ),
+                  for (final item in [
+                    (
+                      serverAnalysis.white.inaccuracies.toString(),
+                      'Inaccuracies',
+                      serverAnalysis.black.inaccuracies.toString()
+                    ),
+                    (
+                      serverAnalysis.white.inaccuracies.toString(),
+                      'Mistakes',
+                      serverAnalysis.black.inaccuracies.toString()
+                    ),
+                    (
+                      serverAnalysis.white.inaccuracies.toString(),
+                      'Blunders',
+                      serverAnalysis.black.inaccuracies.toString()
+                    ),
```

Comment:
We need new translations here, though I have no idea how to add them.

Reply:
I think we already have them, they should be:

```
Text(context.l10n.nbInaccuracies(data.inaccuracies))
Text(context.l10n.nbMistakes(data.mistakes))
Text(context.l10n.nbBlunders(data.blunders))
```

Reply:
This is fine for now.  It can be added later.

Reply:
One trick could be to use the translation with the number and then do `translatedString.replaceAll('\d\', '').trim()`